components/css/partials/_accordion.scss

```diff
@@ -32,5 +41,29 @@
 				content:"\f0fe";
 			}
 		}
+		@each $color,$pair in $colorPairs-accessible{
+			&-#{$color}{
+				background-color:nth($pair,1);
+				color:nth($pair,2);
+			}
+		}
+	}
+	&_list{
+		margin:0;
+		padding:0 20px;
+		list-style-type:none;
+	}
+	&_item{
```

Comment:
I'd suggest `&_listItem` for clarity.

Reply:
Somewhat similar to my comment above, this class is meant to be applied to more than just a `<li>` element. It could belong to a `<p>` as well if that was appropriate. I just wanted this to symbolize a single item within the "list" of the accordion. I'm willing to have a rename though.

Reply:
Do we need to include a style for a <p> if we don't use the <li>?

Reply:
Well that didn't show, do we need to include a style for a p if we don't use an li?

Reply:
@jengarza that was updated in the latest commit. Check out the different variations on the test page:
https://dev.www.purdue.edu/pmmtest/templates/testing/accordion/templates/patterns/accordion.html